features/build_runner/build_runner_output_row.rb

```diff
@@ -3,7 +3,7 @@ module FastlaneCI
   # meaning it's a single message that we want to show to the user
   # and store as a build artifact as part of the build log
   class BuildRunnerOutputRow
-    BUILD_FAIL_TYPES = [:user_error, :build_error, :crash, :shell_error, :build_failure, :test_failure, :abort].to_set
+    BUILD_FAIL_TYPES = [:user_error, :build_error, :crash, :error, :shell_error, :build_failure, :test_failure, :abort].to_set
```

Comment:
@KrauseFx I noticed that `UI.error` in a Fastfile always triggers `:error` for a fail type, so I added that here. Is this something we just missed, or do we expect it to be something else?